supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc

```diff
@@ -72,6 +72,28 @@ Specify the system architecture of your cluster, such as `s390x` or `x86_64`.
 
 *See also*:
 
+[[scale-out]]
+==== image:images/yes.png[yes] scale out (verb)
+*Description*: To _scale-out_is to improve the ability of a cluster to do work by adding more machines. _Scaling out_ refers to increasing the amount of load that the system can handle at one time by increasing the number of servers doing work.
```

Comment:
@lpettyjo @laubai Updated description to not use a hyphen. Let me know if that is incorrect

Reply:
"scale-out storage" would be used to describe storage that scales out rather than up
"scale out" makes sense here because it's the verb